rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -3258,6 +3275,11 @@ namespace rsx
 		flip_status = CELL_GCM_DISPLAY_FLIP_STATUS_DONE;
 		m_queued_flip.in_progress = false;
 
+		if (g_cfg.video.vblank_loop)
+		{
+			post_vblank_event(rsx::uclock());
```

Comment:
This feels like it should just be part of "infinite" vblank mode and does not need the extra loop option